pkg/epp/scheduling/framework/plugins/multi/prefix/plugin.go

```diff
@@ -257,7 +257,7 @@ func (p *Plugin) matchLongestPrefix(ctx context.Context, hashes []BlockHash) map
 }
 
 // hashPrompt divides the prompt into blocks and calculate the prefix cache for each block.
-// hash(0) is the hash of the model name, since different models generally don't share prefix cache.
+// hash(0) is the hash of the model name and cache_salt(if provided), since different models generally don't share prefix cache.
```

Comment:
```suggestion
// hash[0] is calculated including the model name and cache_salt(if provided), since different models generally don't share prefix cache.
```